update review updates to be able to edit in map

meomancer opened the pull request:
This PR is about handling review that is in review section

When in profile:
![Selection_305](https://user-images.githubusercontent.com/4530905/61358486-99cd9200-a8a4-11e9-8c6b-0d0413ebc8c7.png)

when click a review, it will directly into map
showing the form
![Selection_307](https://user-images.githubusercontent.com/4530905/61358581-d3060200-a8a4-11e9-930e-60b62c6a6acb.png)

We can discard by using bottom-right
After changes, when save, and success, it will be pushed to peding
![Selection_308](https://user-images.githubusercontent.com/4530905/61358619-eb761c80-a8a4-11e9-9ee1-944428635b57.png)
![Selection_310](https://user-images.githubusercontent.com/4530905/61358623-ed3fe000-a8a4-11e9-83ca-8c99161ed98d.png)

NOTE:
- This review is saved when user save a data by using others osm_user
- If it is saved using it's own osm (example from map) it will not save into review because the response is already returns the message

meomancer commented:
Updates:
Handling duplication

When duplication happened, when user add in HS.io map or from review in their profile because of odk, user will be asked tor resolve it.

From profile:
![Selection_317](https://user-images.githubusercontent.com/4530905/61540323-9a625600-aa67-11e9-847d-a3a4a91de39c.png)

When click resolve on map, it will force user to map and there will be modal (this will be activate when user create healthsite in HS.io map too

![Selection_318](https://user-images.githubusercontent.com/4530905/61540351-a9e19f00-aa67-11e9-9939-298efcf8e57d.png)

There are 4 choices:
![Selection_318](https://user-images.githubusercontent.com/4530905/61540399-bebe3280-aa67-11e9-98a4-8121139237fe.png)
![Selection_319](https://user-images.githubusercontent.com/4530905/61540401-c087f600-aa67-11e9-9411-3fdd454d5f15.png)
![Selection_320](https://user-images.githubusercontent.com/4530905/61540409-c2ea5000-aa67-11e9-8d47-7ed85a8e13fc.png)
![Selection_326](https://user-images.githubusercontent.com/4530905/61540418-c7166d80-aa67-11e9-96e1-0a16d26496f0.png)

First choice will auto save by forcing add that new one
![Selection_322](https://user-images.githubusercontent.com/4530905/61540452-dbf30100-aa67-11e9-9cec-64893f3d22c4.png)

Second is continue editing new one
![Selection_323](https://user-images.githubusercontent.com/4530905/61540478-e4e3d280-aa67-11e9-9ea0-8e281a159b67.png)

Third will using original one
![Selection_324](https://user-images.githubusercontent.com/4530905/61540513-f200c180-aa67-11e9-9d68-ae0c9f1b0855.png)

The forth one will replace all attributes from duplicate but also copyging tag that not on duplicate buut on original
![Selection_327](https://user-images.githubusercontent.com/4530905/61540563-093faf00-aa68-11e9-8d32-9f2d348a4d26.png)

After that we can do save

timlinux commented:
"If this is duplicate and you rather" -> "If this is a duplicate and you would rather"

Looking really great @meomancer 

Side issue but can you fix the order of attributes display (and in edits form) so it is more logical? 

* Name
* Amenity
* Healthcare
* Healthcare Amenity Type

And generally group things logically e.g. staff count items together

Well done for whipping this up at such short notice!

Will you be able to have it deployed to live soon?

Also just reassure me those test records are not against the live OSM - they already complained when we added a test record in live so we must be ultra-careful to not do that when testing stuff.

Thanks!

Regards

Tim

meomancer commented:
hi @timlinux 
updated, and also
```
Also just reassure me those test records are not against the live OSM - they already complained when we added a test record in live so we must be ultra-careful to not do that when testing stuff.
```
i was using master.osm as my test platform
